aten/src/ATen/native/cuda/DilatedMaxPool2d.cu

```diff
@@ -76,33 +165,10 @@ __global__ void MaxPoolBackward(const int nthreads, const scalar_t* top_diff,
     int h = index/width;
     int w = index - h * width;
 //get some templating performance benefits without actually templating
```

Comment:
nit: now we removed this fake templating thing, we might as well remove this comment that no long stands :)